app/guide/manifest.json

```diff
@@ -0,0 +1,163 @@
+{
+  "title": "Buyer’s Guide",
+  "description": "Read the following information before using the Buying Catalogue. It’ll help you find the information you need to make an informed decision when procuring.",
+  "subtext": [
+    "All products and services listed on the Buying Catalogue are known as Catalogue Solutions. Each Solution has a certain number of Capabilities which can address business needs in primary care.",
+    "To prove these Capabilities, Suppliers must demonstrate that their Catalogue Solution can satisfy Epics which detail the functions required to successfully achieve a Capability.",
+    "There is a link to the full Catalogue Buyer’s Guide at the bottom of this page.",
+    "These are the key terms you need to understand to use the Buying Catalogue effectively:"
+  ],
+  "sections": [
+    {
+      "title": "Catalogue Solution",
+      "description": "A Catalogue Solution is a product or service that has successfully demonstrated compliance with Capabilities and Standards that apply to it. The Catalogue Solution will only be made available on the Buying Catalogue once it has demonstrated its compliance.",
+      "contents": [
+        {
+          "title": "Associated Services",
+          "description": ["Associated Services help you to implement or optimise the live operation of a Catalogue Solution and/or Additional Services, for example, training."]
+        },
+        {
+          "title": "Additional Services",
+          "description": [
+            "Additional Services are add-ons that provide additional Capabilities to a Catalogue Solution for an extra cost.",
+            "When you view Catalogue Solutions on the Buying Catalogue, if there are Additional Services available, information about their Capabilities and price will be presented to you.",
+            "Additional Services may also have Associated Services"
+          ]
+        }
+      ]
+    },
+    {
+      "title": "Capabilities",
+      "description": "",
```

Comment:
I think we discussed this the other day. Do you think it would be worth making description an array and then we shouldn't need all the contents stuff below